lib/auth/init.go

```diff
@@ -1405,6 +1408,15 @@ func applyResources(ctx context.Context, service *Services, resources []types.Re
 			err = service.ClusterConfiguration.SetClusterNetworkingConfig(ctx, r)
 		case types.AuthPreference:
 			err = service.ClusterConfiguration.SetAuthPreference(ctx, r)
+		case interface{ Unwrap() types.Resource153 }:
+			switch r := r.Unwrap().(type) {
+			case *machineidv1pb.Bot:
+				_, err := machineidv1.UpsertBot(ctx, service, r)
+				if err != nil {
+					return trace.Wrap(err)
+				}
```

Comment:
Let err be handled below, like the others?

```suggestion
				_, err = machineidv1.UpsertBot(ctx, service, r)
```

Reply:
effb495686